doc/source/dev/roadmap.rst

```diff
@@ -50,13 +50,16 @@ requested in this respect.
 Statistics enhancements
 -----------------------
 
-The `scipy.stats` enhancements listed in the :ref:`scipy-roadmap-detailed` are of
-particularly high importance to the project.
-
-- Improve the options for fitting a probability distribution to data.
-- Expand the set of hypothesis tests.  In particular, include all the basic
-  variations of analysis of variance.
-- Add confidence intervals for all statistical tests.
```

Comment:
Removing enhancements, as they don't seem quite as important as essential maintenance below.

Reply:
I would keep the note on confidence intervals. I would even maybe mention power?

And while I am there, add a note on improving our doc on how to use all of that. What to use when, what are the limits, etc.

Reply:
I think CI are of top importance, too, but I think we'll have a hard time getting everyone to agree on priorities, so I drew a line between maintainance and enhancements. I can add power to the list in the detailed roadmap, though.

I can add something about documentation here - fixing the defects, like adding what is not currently documented (e.g. missing tutorials, how vectorization works in the context of stats). I don't think we (stats maintainers) have a shared, actionable vision for documentation enhancements, though, so I hesitate to put it documentation enhancements at the top level roadmap.

Reply:
Ok I see your reasoning. To me we did not need to agree on how to do things to list points in there. This is mostly a doc to tell people what we are up to or want to discuss.

Reply:
Ok. I guess could leave "documentation improvements" vague then. But that is only a little more actionable than "code improvements" : P